elyra/tests/pipeline/resources/components/bash_operator.py

```diff
@@ -0,0 +1,162 @@
+#
+# Copyright 2018-2022 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
We should indicate where this source code is coming from and mention whether it was modified (e.g. if changes were made to make the Elyra linters pass)

Reply:
Good idea - I'll add that now! (because changes were made to pass `black` formatting :) )